src/_designs/screens/dashboard/index.vue

```diff
@@ -25,17 +37,38 @@ export default {
     PageHeader,
     Generations,
     Generation,
-    BreadCrumbs
+    BreadCrumbs,
+    TxList,
+    TXListItem
+  },
+  data () {
+    return {
+      transactionsList: []
+    }
   },
   computed:
     mapState('blocks', [
-      'generations'
+      'generations',
+      'height'
     ]),
   mounted: async function () {
     await this.$store.dispatch('blocks/getLatestGenerations', 5)
+    await this.$store.dispatch('blocks/height')
+    await this.fetchTx()
+  },
+  methods: {
+    fetchTx: async function () {
+      try {
+        this.transactionsList = await this.$store.dispatch('transactions/getTxByGeneration', {
+          start: this.height - 10,
```

Comment:
Maybe the following can be done in another PR:
`getTxByGeneration` should take a `limit` parameter. If there are 1000 transactions in the last 10 generations, it is not good to show them all in just one screen.